futurex_openedx_extensions/helpers/models.py

```diff
@@ -306,10 +306,11 @@ class DataExportTask(models.Model):
 
     filename = models.CharField(max_length=255)
     view_name = models.CharField(max_length=255)
+    related_id = models.CharField(max_length=255, null=True, blank=True)
```

Comment:
I'm unsure if it's helpful to have both `blank` and `null`, I think one of them is needed.

@shadinaif @tehreem-sadat does indexing aid search in this field?

```suggestion
    related_id = models.CharField(max_length=255, null=True)
```

Reply:
blank=True will help to add/update db model entry from django-admin.